examples/linear_model/plot_logistic_multinomial_mnist.py

```diff
@@ -0,0 +1,71 @@
+"""
```

Comment:
Maybe this filename should be renamed to: `plot_sparse_logistic_regression_mnist.py` to highlight the sparsity inducing penalty which is quite unusual for logistic regression.